src_modules/voice_commands/index.ts

```diff
@@ -93,17 +91,32 @@ export default class VoiceCommandsModule extends Module {
         this.registerConfigEntry(this.tokenConfigEntry);
 
         this.voiceCommandsEnabledConfigEntry = new BooleanGuildConfigEntry({
-            name: "voiceCommandsEnabled",
-        }, bot.database, true);
-        this.registerConfigEntry(this.voiceCommandsEnabledConfigEntry);
+                    name: "voiceCommandsEnabled",
+                }, bot.database, true);
+                this.registerConfigEntry(this.voiceCommandsEnabledConfigEntry);
+        
+        this.autoJoinEnabledConfigEntry = new BooleanGuildConfigEntry({
+                    name: "autoJoinEnabled",
+                }, bot.database, false);
+                this.registerConfigEntry(this.autoJoinEnabledConfigEntry);
 
 
         bot.intents.push(GatewayIntentBits.GuildVoiceStates);
 
         this.client = new VoiceBackendClient(this);
         this.voiceCommands = new VoiceCommands(this, bot);
-        this.phoneticCache = new PhoneticCache(this, bot);
-        this.speechCache = new SpeechCache(this, bot);
+                this.phoneticCache = new PhoneticCache(this, bot);
+                this.speechCache = new SpeechCache(this, bot);
+        
+        this.registerCommandGroup({
```

Comment:
This is not how you create a command group. Create a new CommandGroup, and add it as a proprty. See this example:

this.musicCommand = new CommandGroup(
    {
        allowedPrivileges: ["everyone"],
        author: "extcord",
        description: "Play music",
        name: "music",
    },
);
this.musicCommand.addSubcommands(
    new PauseCommand(this),
    new PlayCommand(this),
    new ResumeCommand(this),
    new SkipCommand(this),
    new StopCommand(this),
    new VolumeCommand(this),
    new QueueCommand(this),
    new LyricsCommand(this),
    new ClearCommand(this),
    new PopCommand(this),
    new ShuffleCommand(this),
);
this.musicCommand.addPhrases(...phrases);
this.registerCommand(this.musicCommand);

Reply:
Add the missing CommandGroup import from "../..". Also property this.voiceCommandsGroup is missing, add the definition. You also forgot to register the phrases to the command group.